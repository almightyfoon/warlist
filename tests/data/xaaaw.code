c8101b_-0jmPmKmJmI54mUmUjXmSmj5wmj5wmG___-0p4a4QjFjF4H4Hg4g45gfI5e5e5fi7
