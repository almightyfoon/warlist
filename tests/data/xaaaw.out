https://conflictchamber.com/#c8101b_-0jmPmKmJmI54mUmUjXmSmj5wmj5wmG___-0p4a4QjFjF4H4Hg4g45gfI5e5e5fi7

Circle Army - 75 / 75 points
[Theme] The Devourer's Host

[Iona 1] Iona the Unseen [+28]
 - Ghetorix [19]
 - Warpwolf Stalker [17]
Lord of the Feast [0(6)]
Tharn Ravager Shaman [4]
Tharn Wolf Rider Champion [0(7)]
Tharn Wolf Rider Champion [0(7)]
Boil Master & Spirit Cauldron [5]
Br√≠ghid & Caul [7]
Tharn Ravagers (max) [15]
 - Tharn Ravager Chieftain [5]
Tharn Ravagers (max) [15]
 - Tharn Ravager Chieftain [5]
Tharn Wolf Riders (min) [11]

Circle Army - 75 / 75 points
[Theme] The Bones of Orboros

[Krueger 2] Krueger the Stormlord [+27]
 - Megalith [20]
 - Woldwarden [14]
 - Woldwarden [14]
 - Woldwyrd [9]
 - Woldwyrd [9]
Blackclad Stoneshaper [0(3)]
Blackclad Stoneshaper [0(3)]
Blackclad Wayfarer [0(4)]
Bog Trog Mist Speaker [4]
Sentry Stone & Mannikins [5]
Sentry Stone & Mannikins [5]
Shifting Stones [3]
Celestial Fulcrum [19]
